templated_files.mk

```diff
@@ -10,6 +10,7 @@ $(eval $(call mldb_install_templated_directory,mldb/container_files/public_html/
 
 $(eval $(call mldb_install_templated_file,mldb/container_files/init/05-mldb-id-mapping.sh,$(ETC)/my_init.d/05-mldb-id-mapping.sh,555))
 $(eval $(call mldb_install_templated_file,mldb/container_files/init/mldb_runner.sh,$(ETC)/service/mldb_runner/run,555))
+$(eval $(call mldb_install_templated_file,mldb/container_files/mldb.conf,$(ETC)/service/mldb_runner/run,664))
```

Comment:
I think we could put `mldb.conf`  directly in `/etc`  instead of deep inside the `runit` directory.

Minor nit:  0644.